### ISSUE TITLE
Working area not expand to window size

### ISSUE THREAD

**Issue by mawib** [ONLYOFFICE/DesktopEditors#694]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

ONLYOFFICE Desktop Editors version 6.3.1.56

### Expected Behavior

When open OnlyOffice I expect the window can be resized and working area will be used in the whole window.
This works on a second computer I own with a 1920x1080 screen. (POP OS 21.04).

### Actual Behavior

Please se image.
![image](https://user-images.githubusercontent.com/50695493/127654301-8e804341-2971-4ea3-9113-f0687ae69e95.png)
Working area fixed in left top corner and are stuck. Can't resize as the rest of the window. This happes on my laptop with a 3840x2160 resolution.

### Reproduction Steps

Install DesktopEditors on a Linux computer with a resolution at 3840x2160 and problem shows up.

### Additional information

             ...-:::::-...                  
          .-MMMMMMMMMMMMMMM-.                                             ------------------ 
      .-MMMM`..-:::::::-..`MMMM-.                                                      OS: Linux Mint 20.2 x86_64 
    .:MMMM.:MMMMMMMMMMMMMMM:.MMMM:.                        Host: DELL XPS 13 9370 
   -MMM-M---MMMMMMMMMMMMMMMMMMM.MMM-                Kernel: 5.4.0-80-generic 
 `:MMM:MM`  :MMMM:....::-...-MMMM:MMM:`                                 Uptime: 1 day, 5 hours, 25 mins 
 :MMM:MMM`  :MM:`  ``    ``  `:MMM:MMM:                                     Packages: 2229 (dpkg), 18 (flatpak) 
.MMM.MMMM`  :MM.  -MM.  .MM-  `MMMM.MMM.                         Shell: bash 5.0.17 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM-MMM:                         Resolution: 3840x2160 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM:MMM:                          DE: MATE 
:MMM:MMMM`  :MM.  -MM-  .MM:  `MMMM-MMM:                         WM: Metacity (Marco) 
.MMM.MMMM`  :MM:--:MM:--:MM:  `MMMM.MMM.                         WM Theme: Mint-Y-Dark 
 :MMM:MMM-  `-MMMMMMMMMMMM-`  -MMM-MMM:                  Theme: Mint-Y-Dark [GTK2/3] 
  :MMM:MMM:`                `:MMM:MMM:                                            Icons: Mint-Y-Dark [GTK2/3] 
   .MMM.MMMM:--------------:MMMM.MMM.                                      Terminal: mate-terminal 
     '-MMMM.-MMMMMMMMMMMMMMM-.MMMM-'                        Terminal Font: Monospace 14 
       '.-MMMM``--:::::--``MMMM-.'                                                       CPU: Intel i7-8550U (8) @ 4.000GHz 
            '-MMMMMMMMMMMMM-'                                                    GPU: Intel UHD Graphics 620 
               ``-:::::-``                                                                                Memory: 3526MiB / 15712MiB

**Comment by ShockwaveNN** [comment]
@mawib Hi, could you try to install on same system not deb package, but flatpak or snap of DesktopEditors and see if problem stays same

We saw several reports of similar issue, but never reproduced it on our end (

**Comment by mawib** [comment]
I try and return with an update.

**Comment by mawib** [comment]
Same behaviour on .deb AppImage, Snap and Flatpak.

**Comment by ShockwaveNN** [comment]
Ok. forgot snap\appimage or flatpak

Could you run

```
desktopeditors --force-scale=1
```

or

```
desktopeditors --force-scale=2
```

and see if those variants looks good for you?

**Comment by mawib** [comment]
No, scaling works but there are still a big grey area around the working area

scale1
![image](https://user-images.githubusercontent.com/50695493/127732613-d0b6d456-ff97-4125-be39-2d3ade8b0960.png)

scale2
![image](https://user-images.githubusercontent.com/50695493/127732638-af5ef01d-efda-493a-8d51-f3cea4c5c1c2.png)

**Comment by ShockwaveNN** [comment]
We got several 4k displays at our disposal, but never able to reproduce this problem, although users are reported similar problems before. Maybe problem is actual only for 13 4k displays, but we have no such small screens

My last idea - try to delete config file
```
rm ~/.config/onlyoffice/DesktopEditors.conf
```
Maybe this helps

**Comment by mawib** [comment]
File deleted. After re-open DesktopEditors a new file is created.
Content are still:

General]
maximized=false
position=@Rect(0 0 1920 1053)
titlebar=custom

But UI looks the same as before.

**Comment by ShockwaveNN** [comment]
@mawib Now I'm out of ideas, I'm sure this some kind a bug, but we cannot reproduce it at all

I'l leave this issue opened until someone have some idea how to reproduce this problem on our systems, maybe some display config tinkering

**Comment by mawib** [comment]
Ok. Thank you for your help. :-)

**Comment by levin-kelevra** [comment]
version 7.0.0.127 - same behaviour

**Comment by levin-kelevra** [comment]
`$ env QT_SCREEN_SCALE_FACTORS=1 QT_SCALE_FACTOR=1 QT_FONT_DPI=263 PLASMA_USE_QT_SCALING=1 /usr/bin/onlyoffice-desktopeditors %U`
and it works fine

**Comment by mawib** [comment]
For your information. I installed Manjaro with Gnome v 41.2 and scaling works without any additional settings.  I tried with both 6.x.x version and the new 7.0 version of OnlyOffice.

██████████████████  ████████   magnusw@magnusw-xps 
██████████████████  ████████   ------------------- 
██████████████████  ████████   OS: Manjaro Linux x86_64 
██████████████████  ████████   Host: XPS 13 9370 
████████            ████████   Kernel: 5.15.12-1-MANJARO 
████████  ████████  ████████   Uptime: 2 days, 28 mins 
████████  ████████  ████████   Packages: 1382 (pacman), 20 (flatpak) 
████████  ████████  ████████   Shell: zsh 5.8 
████████  ████████  ████████   Resolution: 3840x2160 
████████  ████████  ████████   DE: GNOME 41.2 
████████  ████████  ████████   WM: Mutter 
████████  ████████  ████████   WM Theme: Adwaita-maia-compact-dark 
████████  ████████  ████████   Theme: Adwaita-maia-compact-dark [GTK2/3] 
████████  ████████  ████████   Icons: Papirus-Dark-Maia [GTK2/3] 
                               Terminal: gnome-terminal 
                               CPU: Intel i7-8550U (8) @ 4.000GHz 
                               GPU: Intel UHD Graphics 620 
                               Memory: 4165MiB / 15701MiB

**Comment by ShockwaveNN** [comment]
@mawib So to be clear

Right now problem with scale is actual for MATE and not actual to Gnome with same version of System and ONLYOFFICE DesktopEditors?

**Comment by mawib** [comment]
Yes, I beleve so. I haven't have this problem after moving over to Gnome. My machine have temporary being installed with Windows 11 since Mint MATE  for a time, but now I run Manjaro. I can try install newest version of Mint MATE on a VM and see if I can replicate the issue.

**Comment by ShockwaveNN** [comment]
@mawib Please do that if you got free time, it will gladly help us to localize problem

**Comment by mawib** [comment]
OK I have installed latest version of Mint Mate 20.3 on a QEMU/KVM vm and set resolution to 3840 x 2160. 
If I use 100% scaling on the system - scaling works as designed in OnlyOffice. Both version 6 and 7.
If I use 200% scaling on the system - scaling in OnlyOffice looks like above pictures.

I use 200% scaling on Manjaro with Gnome and OnlyOffice works good when scaling inside the application.

**Comment by ShockwaveNN** [comment]
Ok thanks! We'll have a look into it

**Comment by siddhpant** [comment]
I have set `QT_SCALE_FACTOR=1.25` and I face this issue. I'm on GNOME.

Setting `QT_SCALE_FACTOR` to 1 solves the issue.

**Comment by filippogiad** [comment]
Dear All, I find the same exact issue in Windows 11 with a 2880x1800 display.
![Screenshot 2024-12-10 181348](https://github.com/user-attachments/assets/81fb43dd-48dd-4518-b2ce-01ccf7285737)
